integreat_cms/cms/templates/dashboard/_dummy_widget.html

```diff
@@ -1,18 +0,0 @@
-{% extends "_collapsible_box.html" %}
-
-{% load i18n %}
-{% load static %}
-
-{% block collapsible_box_icon %}
-    eye
-{% endblock %}
-
-{% block collapsible_box_title %}
-    {% trans 'At a glance' %}
-{% endblock %}
-
-{% block collapsible_box_content %}
-    <div class="p-3 w-full">
-        <!-- TODO -->
```

Comment:
We can add a text like "Upcoming feature" in here to make it easier to understand why it's there.

Reply:
This is just for the admins and I guess that we'll have some changes to the dashboard as well. Therefore, I prefer to just get rid of it :)